lib/validators/lib/andOrValidatorProcessor.js

```diff
@@ -0,0 +1,98 @@
+const register = require('../../register')
+const { getValidatorPromises } = require('../../flex')
+const consolidateResult = require('../options_processor/options/lib/consolidateResults')
+const constructErrorOutput = require('../options_processor/options/lib/constructErrorOutput')
+
+const ERROR_MESSAGE = `Failed to validate because the 'validate' option is missing or empty. Please check the documentation.`
+
+const andOrValidatorProcessor = async (context, settings, registry, validatorName) => {
+  const validatorContext = { name: validatorName }
+
+  if (!Array.isArray(settings) || settings.length === 0) {
+    return consolidateResult(
+      [
+        constructErrorOutput(
+          validatorName,
+          '',
+          settings,
+          ERROR_MESSAGE
+        )
+      ],
+      validatorContext
+    )
+  }
+
+  const rules = { validate: settings }
+
+  try {
+    register.registerValidators(rules, registry)
+  } catch (err) {
+    // Should I avoid a console error and consolidateOutput instead?
```

Comment:
Depends. If you want to know how often this happens, I'd do `log.info`

Reply:
removed the comments